boards/catie/zest_core_stm32l4a6rg/sixtron_connector.dtsi

```diff
@@ -54,34 +54,36 @@
                    <DIO20 0 &gpiob 9 0>;
     };
 
-    sixtron_connector_1_pwms: sixtron-connector-1-pwms {
-    	compatible = "pwm-leds";
+    sixtron_connector_1_pwm: sixtron-connector-1-pwm {
+        compatible = "sixtron-pwm";
+        #pwm-cells = <3>;
+        pwm-map-mask = <0xffffffff 0x0 0x0>;
+        pwm-map-pass-thru = <0x0 0xffffffff 0x0f>;
+        pwm-map = <PWM1 0 0 &pwm1 3 0 0>,	/* PWM1 = */
+                  <PWM2 0 0 &pwm1 2 0 0>,	/* PWM2 = */
+                  <PWM3 0 0 &pwm3 3 0 0>;	/* PWM3 = */
```

Comment:
Please remove comments here.

Reply:
Resolved in [94e7aac](https://github.com/catie-aq/zephyr_zest-core-stm32l4a6rg/pull/48/commits/94e7aacb3330fa2b14a405f562ce9d163bdae223).